pytorch/inference/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -181,6 +181,9 @@ RUN pip install --no-cache-dir -U \
     boto3 \
     scipy \
     opencv-python \
+    # "nvgpu" is a dependency of TS but is disabled in SM DLC build, via ENV Variable
+    # due to incompatibility with SM hosts 
```

Comment:
What is the ENV on this dockerfile that disables `nvgpu`?

Reply:
`ENV TS_DISABLE_SYSTEM_METRICS=true` at line 282 for the GPU docker image (SageMaker section)

Reply:
Thanks! Could you please add this in the comment on the dockerfile?